ts/src/huobi.ts

```diff
@@ -2809,120 +2898,126 @@ export default class huobi extends Exchange {
+                const canWithdraw = this.safeValue (chain, 'we');
+                withdrawEnabled = canWithdraw ? true : withdrawEnabled;
+                currencyActive = (canDeposit && canWithdraw) ? true : currencyActive;
+                const networkCode = super.networkIdToCode (networkTitle);
+                const precision = this.parsePrecision (this.safeString (chain, 'wp'));
+                minPrecision = (minPrecision === undefined) ? precision : Precise.stringMin (minPrecision, precision);
                 networks[networkCode] = {
-                    'info': chainEntry,
-                    'id': uniqueChainId,
+                    'id': networkId,
                     'network': networkCode,
+                    'active': (canDeposit && canWithdraw),
+                    'deposit': canDeposit,
+                    'withdraw': canWithdraw,
+                    'fee': undefined,
+                    'precision': this.parseNumber (precision),
                     'limits': {
+                        'deposit': {
+                            'min': this.safeNumber (chain, 'dma'),
+                            'max': undefined,
+                        },
                         'withdraw': {
-                            'min': minWithdraw,
-                            'max': maxWithdraw,
+                            'min': this.safeNumber (chain, 'wma'),
+                            'max': undefined,
                         },
                     },
-                    'active': active,
-                    'deposit': depositEnabled,
-                    'withdraw': withdrawEnabled,
-                    'fee': fee,
-                    'precision': this.parseNumber (precision),
+                    'depositTagNeeded': this.safeValue (chain, 'adt'),
+                    'isDefault': isDefaultChain,
+                    'info': chain,
                 };
             }
+            const type = isFiat ? 'fiat' : 'crypto';
             result[code] = {
-                'info': entry,
+                'info': chains,
                 'code': code,
                 'id': currencyId,
+                'name': undefined,
                 'active': currencyActive,
-                'deposit': deposit,
-                'withdraw': withdraw,
+                'deposit': depositEnabled,
+                'withdraw': withdrawEnabled,
+                'type': type,
                 'fee': undefined,
-                'name': undefined,
+                'precision': this.parseNumber (minPrecision),
                 'limits': {
                     'amount': {
                         'min': undefined,
                         'max': undefined,
                     },
-                    'withdraw': {
-                        'min': minWithdraw,
-                        'max': maxWithdraw,
-                    },
                 },
-                'precision': this.parseNumber (minPrecision),
                 'networks': networks,
```

Comment:
@ttodua is this expected? Apparently we have 11 different chains but in the unified object there's only 4
![image](https://github.com/ccxt/ccxt/assets/43336371/e88674ad-0707-463f-9355-5f526219a30c)

Reply:
let me check this too.